plugins/GSdx/GLLoader.cpp

```diff
@@ -385,7 +385,7 @@ namespace GLLoader {
 
 		if (glGetStringi && max_ext) {
 			for (GLint i = 0; i < max_ext; i++) {
-				string ext((const char*)glGetStringi(GL_EXTENSIONS, i));
```

Comment:
What is the advantage to use `{...}` instead of `( ... )` for the constructor ? (I'm just curious)

Reply:
It avoids ambiguity in some cases - what might look like a variable declaration might actually be a function declaration. Some further reading at https://herbsutter.com/2013/05/09/gotw-1-solution/ (or you can look up "most vexing parse")

Though in this case, I felt like changing it, got part of the way through, and then abandoned the effort...

Reply:
Thanks for the info.